### PR TITLE
Lib/Trk_minnow_dolphin work

### PR DESCRIPTION
**Pull request by Yotona**

### CODE REVIEW

File: libs/TRK_MINNOW_DOLPHIN/GCN/EXI2_DDH_GCN/main.c [hunk 1]
```diff
@@ -116,7 +90,7 @@ COMPILER_STRIP_GATE(0x803A2D50, &lit_320);
 #pragma pop
 
 /* 804519C0-804519C8 000EC0 0004+04 3/3 0/0 0/0 .sbss            gIsInitialized */
-static u8 gIsInitialized[4 + 4 /* padding */];
+static BOOL gIsInitialized[1 + 1 /* padding */];
```
Comment (TakaRikka) [inline libs/TRK_MINNOW_DOLPHIN/GCN/EXI2_DDH_GCN/main.c:93]: is the padding still needed or can this be just `static BOOL gIsInitialized;` ?

File: libs/TRK_MINNOW_DOLPHIN/GCN/EXI2_GDEV_GCN/main.c [hunk 2]
```diff
@@ -116,7 +101,7 @@ COMPILER_STRIP_GATE(0x803A2E30, &lit_320);
 #pragma pop
 
 /* 804519C8-804519D0 000EC8 0004+04 3/3 0/0 0/0 .sbss            gIsInitialized */
-static u8 gIsInitialized[4 + 4 /* padding */];
+static s32 gIsInitialized[1 + 1 /* padding */];
```
Comment (TakaRikka) [inline libs/TRK_MINNOW_DOLPHIN/GCN/EXI2_GDEV_GCN/main.c:104]: same question about removing padding here

File: libs/TRK_MINNOW_DOLPHIN/MetroTRK/Portable/msghndlr.c [hunk 3]
```diff
@@ -268,7 +268,7 @@ asm s32 TRKDoReset(TRKBuffer*) {
 
 /* ############################################################################################## */
 /* 8044F288-8044F290 07BFA8 0004+04 4/4 0/0 0/0 .bss             IsTRKConnected */
-SECTION_BSS static u8 IsTRKConnected[4 + 4 /* padding */];
+SECTION_BSS static BOOL IsTRKConnected[1 + 1 /* padding */];
```
Comment (TakaRikka) [inline libs/TRK_MINNOW_DOLPHIN/MetroTRK/Portable/msghndlr.c:271]: same question about removing padding here

File: libs/TRK_MINNOW_DOLPHIN/MetroTRK/Portable/nubinit.c [hunk 4]
```diff
@@ -45,23 +45,19 @@ asm void TRKNubWelcome(void) {
...
 /* ############################################################################################## */
 /* 803A26A4-803A26B8 02ED04 0010+04 1/1 0/0 0/0 .rodata          @154 */
 SECTION_RODATA static char const lit_154[] = "Initialize NUB\n";
 COMPILER_STRIP_GATE(0x803A26A4, &lit_154);
 
 /* 8044D8B8-8044D8C0 07A5D8 0004+04 1/1 4/4 0/0 .bss             gTRKBigEndian */
-SECTION_BSS extern u8 gTRKBigEndian[4 + 4 /* padding */];
-SECTION_BSS u8 gTRKBigEndian[4 + 4 /* padding */];
+SECTION_BSS extern s32 gTRKBigEndian[1 + 1 /* padding */];
+SECTION_BSS s32 gTRKBigEndian[1 + 1 /* padding */];
```
Comment (TakaRikka) [inline libs/TRK_MINNOW_DOLPHIN/MetroTRK/Portable/nubinit.c:60]: same question about removing padding here

**Review by TakaRikka — COMMENTED** [review]
just a few small things

**Comment by Yotona** [comment]
Each of those was able to just be a static, no padding needed. Thanks!

**Comment by TakaRikka** [comment]
sorry will need to re-pull master for the progress conflict fix. eventually we need to figure out a better system for progress

**Review by TakaRikka — APPROVED** [review]

**Pull request merged**